cgp/hl_api.py

```diff
@@ -9,44 +9,43 @@
 
 
 def evolve(
-    pop: Population,
     objective: Callable[[IndividualBase], IndividualBase],
-    ea: MuPlusLambda,
+    pop: Population = Population(),
+    ea: MuPlusLambda = MuPlusLambda(),
     min_fitness: float = np.inf,
     termination_fitness: float = np.inf,
-    max_generations: int = np.iinfo(np.int64).max,
+    max_generations: int = 1000,
```

Comment:
Not sure this change is a good idea. If the user wants to use `max_objective_calls` this default will still be used (unless overwritten)